src/navigation/HardwareWalletTxNavigator.tsx

```diff
@@ -1,18 +1,110 @@
 import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
-import React from 'react';
+import React, { useCallback, useEffect } from 'react';
 import Routes from '@/navigation/routesNames';
 import { BackgroundProvider } from '@/design-system';
-import { useDimensions } from '@/hooks';
+import { useDimensions, useWallets } from '@/hooks';
 import { PairHardwareWalletAgainSheet } from '@/screens/hardware-wallets/PairHardwareWalletAgainSheet';
 import { PairHardwareWalletErrorSheet } from '@/screens/hardware-wallets/PairHardwareWalletErrorSheet';
 import { SimpleSheet } from '@/components/sheet/SimpleSheet';
+import { useLedgerConnect } from '@/hooks/useLedgerConnect';
+import { LEDGER_ERROR_CODES } from '@/utils/ledger';
+import { useNavigation } from '@/navigation';
+import { logger } from '@/logger';
+import { DebugContext } from '@/logger/debugContext';
+// eslint-disable-next-line no-restricted-imports
+import { RouteProp, useRoute } from '@react-navigation/core';
+import { atom, useRecoilState } from 'recoil';
+import { MMKV } from 'react-native-mmkv';
+
+export const ledgerStorage = new MMKV({
+  id: 'ledgerStorage',
+});
```

Comment:
love it ty

Reply:
one second thought not gonna do this since i need to use the mmkv hook to listen to state for now 

not ideal but its only one value and controlled by a const